panic starting pipeline in prow mode

### Summary 

Creating a new repo fails with:
```
...
Creating GitHub webhook for davidcurrie-org/golang-http-dev for url https://hook.jx.dev.cd-process.com/hook
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xd0 pc=0x2205b12]

goroutine 1 [running]:
github.com/jenkins-x/jx/pkg/prow.(*Options).GetPostSubmitJob(0xc000d014f0, 0xc001104390, 0xf, 0xc0011043a0, 0xf, 0xc0011043b0, 0x6, 0x0, 0x0, 0x0, ...)
	/home/jenkins/go/src/github.com/jenkins-x/jx/pkg/prow/prow.go:535 +0x82
github.com/jenkins-x/jx/pkg/jx/cmd.(*StartPipelineOptions).createProwJob(0xc000d012e0, 0xc001104390, 0x26, 0x0, 0x2)
	/home/jenkins/go/src/github.com/jenkins-x/jx/pkg/jx/cmd/start_pipeline.go:178 +0x12e
github.com/jenkins-x/jx/pkg/jx/cmd.(*StartPipelineOptions).Run(0xc000d012e0, 0xc000738150, 0xc000d012b0)
	/home/jenkins/go/src/github.com/jenkins-x/jx/pkg/jx/cmd/start_pipeline.go:155 +0x38e
github.com/jenkins-x/jx/pkg/jx/cmd.(*ImportOptions).addProwConfig(0xc000290480, 0xc000af9800, 0x36, 0x3b62520, 0xc000f15860)
	/home/jenkins/go/src/github.com/jenkins-x/jx/pkg/jx/cmd/import.go:913 +0x35d
github.com/jenkins-x/jx/pkg/jx/cmd.(*ImportOptions).doImport(0xc000290480, 0x0, 0x0)
	/home/jenkins/go/src/github.com/jenkins-x/jx/pkg/jx/cmd/import.go:884 +0x202
github.com/jenkins-x/jx/pkg/jx/cmd.(*ImportOptions).Run(0xc000290480, 0xc0008bf980, 0x60)
	/home/jenkins/go/src/github.com/jenkins-x/jx/pkg/jx/cmd/import.go:410 +0x3f8
github.com/jenkins-x/jx/pkg/jx/cmd.(*CreateProjectOptions).ImportCreatedProject(0xc000290480, 0xc000a953b0, 0x42, 0x1, 0x1)
	/home/jenkins/go/src/github.com/jenkins-x/jx/pkg/jx/cmd/create.go:116 +0x65
github.com/jenkins-x/jx/pkg/jx/cmd.(*CreateQuickstartOptions).Run(0xc000290480, 0x0, 0x0)
	/home/jenkins/go/src/github.com/jenkins-x/jx/pkg/jx/cmd/create_quickstart.go:218 +0x74b
github.com/jenkins-x/jx/pkg/jx/cmd.NewCmdCreateQuickstart.func1(0xc00062d900, 0x58da808, 0x0, 0x0)
	/home/jenkins/go/src/github.com/jenkins-x/jx/pkg/jx/cmd/create_quickstart.go:88 +0x72
github.com/spf13/cobra.(*Command).execute(0xc00062d900, 0x58da808, 0x0, 0x0, 0xc00062d900, 0x58da808)
	/home/jenkins/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:766 +0x2cc
github.com/spf13/cobra.(*Command).ExecuteC(0xc0002b9680, 0x0, 0x0, 0x0)
	/home/jenkins/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:852 +0x2fd
github.com/spf13/cobra.(*Command).Execute(0xc0002b9680, 0xc0000f2ff0, 0x3b10cc0)
	/home/jenkins/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800 +0x2b
github.com/jenkins-x/jx/cmd/jx/app.Run(0xc000799f88, 0xc0000aa058)
	/home/jenkins/go/src/github.com/jenkins-x/jx/cmd/jx/app/jx.go:17 +0xdc
main.main()
	/home/jenkins/go/src/github.com/jenkins-x/jx/cmd/jx/jx.go:10 +0x22
```

Looks like a regression introduced by the removal of:
https://github.com/jenkins-x/jx/commit/51aada092f4582194edd2fe8750da9d12c95c070#diff-5bf47ea189a45ad840ce340caf62dc99L115

### Steps to reproduce the behavior

`jx create quickstart` against a cluster running in prow mode

### Jx version

The output of `jx version` is:

```
NAME               VERSION
jx                 1.3.699
jenkins x platform 0.0.3110
Kubernetes cluster v1.11.5-gke.5
kubectl            v1.12.0
helm client        v2.11.0+g2e55dbe
helm server        v2.12.1+g02a47c7
git                git version 2.18.0
Operating System   Mac OS X 10.14.2 build 18C54
```

### Kubernetes cluster

`jx create cluster gke`

### Expected behavior

Starts pipeline for new project.

### Actual behavior

Panics.

I got something similar with jx v1.3.700 and likely related, just a different invocation...

```
$ jx start pipeline
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xd0 pc=0x2205421]

goroutine 1 [running]:
github.com/jenkins-x/jx/pkg/prow.(*Options).GetReleaseJobs(0xc000432a50, 0x58da808, 0x0, 0x0, 0x2, 0x0)
	/home/jenkins/go/src/github.com/jenkins-x/jx/pkg/prow/prow.go:508 +0x41
github.com/jenkins-x/jx/pkg/jx/cmd.(*StartPipelineOptions).Run(0xc000432840, 0x0, 0x0)
	/home/jenkins/go/src/github.com/jenkins-x/jx/pkg/jx/cmd/start_pipeline.go:120 +0x188
github.com/jenkins-x/jx/pkg/jx/cmd.NewCmdStartPipeline.func1(0xc00055b180, 0x58da808, 0x0, 0x0)
	/home/jenkins/go/src/github.com/jenkins-x/jx/pkg/jx/cmd/start_pipeline.go:88 +0x72
github.com/spf13/cobra.(*Command).execute(0xc00055b180, 0x58da808, 0x0, 0x0, 0xc00055b180, 0x58da808)
	/home/jenkins/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:766 +0x2cc
github.com/spf13/cobra.(*Command).ExecuteC(0xc00011e780, 0x0, 0x0, 0x0)
	/home/jenkins/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:852 +0x2fd
github.com/spf13/cobra.(*Command).Execute(0xc00011e780, 0xc0000f4d20, 0x3b11060)
	/home/jenkins/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800 +0x2b
github.com/jenkins-x/jx/cmd/jx/app.Run(0xc0007d7f88, 0xc0000ac058)
	/home/jenkins/go/src/github.com/jenkins-x/jx/cmd/jx/app/jx.go:17 +0xdc
main.main()
	/home/jenkins/go/src/github.com/jenkins-x/jx/cmd/jx/jx.go:10 +0x22
```

--- a/pkg/jx/cmd/start_pipeline.go
+++ b/pkg/jx/cmd/start_pipeline.go
@@ -97,7 +97,7 @@ func NewCmdStartPipeline(f Factory, in terminal.FileReader, out terminal.FileWri
 
 // Run implements this command
 func (o *StartPipelineOptions) Run() error {
-	_, err := o.KubeClient()
+	kubeClient, err := o.KubeClient()
 	if err != nil {
 		return err
 	}
@@ -113,6 +113,7 @@ func (o *StartPipelineOptions) Run() error {
 	args := o.Args
 	names := []string{}
 	o.ProwOptions = prow.Options{
+		KubeClient: kubeClient,
 		NS: o.currentNamespace,
 	}
 	if len(args) == 0 {

